codecs/src/aper/decode/decode_charstrings.rs

```diff
@@ -1,9 +1,9 @@
 //! Functionality for decoding character strings
 
+use super::decode_internal::decode_length_determinent;
```

Comment:
A small 'nitpick' in the way I typically use `use`. 

1. First everything from `std`
2. Blank line and then `use` from crates that are our dependency.
3. Blank line and then `use` current `crate` level imports
4. Blank line and then `use` `super` imports.

This is mostly a matter of style but for the reader it's much easier to understand where a particular used definition has come. 

So this line should go below the `crate::` use lines after an empty line as per the convention.